DataTests/DataControllerTests.swift

```diff
@@ -18,10 +18,6 @@ class DataControllerTests: CoreDataTestCase {
         let viewContext = DataController.viewContext
         XCTAssertEqual(try! viewContext.count(for: fetchRequest), 0)
         
-        DataController.perform { context in
-            XCTAssertEqual(try! context.count(for: self.fetchRequest), 0)
```

Comment:
This is a bug that happened in few places in my tests, assert code should not be run on a background thread because the unit test might be finished before this gets called